.github/workflows/Automation Master Legacy Workflow.yml

```diff
@@ -299,25 +310,38 @@ jobs:
         run: |
           echo solutionFilePath=$(find . -type f -name '*.sln') >> $GITHUB_OUTPUT
         shell: bash
-
-      - name: Find or create NuGet.config file
-        if: inputs.owner == 'SkylineCommunications'
-        id: findOrCreateNuGetFile
+    
+      - name: Find branch
```

Comment:
Merge/Rebase main branch in this branch as currently there are conflicts and the PR contains changes that were also done in main.